docs/sections/user_guide/cli/drivers/fv3.rst

```diff
@@ -40,6 +40,8 @@ The ``uw`` mode for configuring and running FV3.
          A run
        runscript
          The runscript
+       validate
+         Validate driver config
```

Comment:
Would you be willing to consider a change to this wording like "Validate the UW driver config"? @NaureenBharwaniNOAA and I are finding that when JEDI enters the mix we are getting some ambiguity around driver configs and component configs, especially when they're both in the same format.

Reply:
Definitely. I was just pairing with @NaureenBharwaniNOAA and realized the same thing.